inject.js

```diff
@@ -346,6 +399,7 @@
         if (inIframe())
           docs.push(window.top.document);
       } catch (e) {
+        // no operation
```

Comment:
I didn't like an open and closed parenthesis as it made me believe something was cut from this section erroneously.